models/marts/core/int_orders__pivot.sql

```diff
@@ -0,0 +1,26 @@
+{%- set payment_methods = ['bank_transfer','coupon','credit_card','gift_card'] -%}
+
+with payments as (
+    select * from {{ ref('stg_payments') }}
+),
+
+pivoted as (
+    select
+        order_id,
+        {% for p in payment_methods -%}

```

Comment:
Using `p` is acceptable here! In the future, I recommend having something more descriptive. It makes your code more readable for those who might be hoping into the project. This goes in line with why we never alias field names to something shorten like `price as p`. Code easily readable to humans will save everyone some brainpower :)